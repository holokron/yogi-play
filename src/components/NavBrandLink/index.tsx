import { Link } from "react-router-dom";

export default function NavBrandLink(): React.ReactElement<{}> {
  return (
    <Link to="/" className="navbar-brand">
      <img
        className="d-inline-block align-top"
        alt="Yogi PLAY"
        width={30}
        src="/icons/icon-64x64.png"
      />{" "}
      Yogi PLAY
    </Link>
  );
}
