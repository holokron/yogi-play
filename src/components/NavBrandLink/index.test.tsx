import { describe, it, expect } from "vitest";
import { render } from "@testing-library/react";
import { MemoryRouter } from "react-router-dom";
import NavBrandLink from ".";

describe("@components/NavBrandLink", () => {
  it("renders correctly", () => {
    const tree = render(
      <MemoryRouter>
        <NavBrandLink />
      </MemoryRouter>
    );

    expect(tree).toMatchSnapshot();
  });
});
