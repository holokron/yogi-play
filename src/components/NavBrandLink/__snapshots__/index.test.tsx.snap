// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@components/NavBrandLink renders correctly 1`] = `
<a
  className="navbar-brand"
  href="/"
  onClick={[Function]}
>
  <img
    alt="Yogi PLAY"
    className="d-inline-block align-top"
    src="/icons/icon-64x64.png"
    width={30}
  />
   
  Yogi PLAY
</a>
`;
